Makefile

```diff
@@ -115,8 +115,32 @@ swagger: swagger/docs.go
 
 .PHONY: update-licenses
 update-licenses:
-	go get github.com/google/go-licenses
-	$(shell echo "Module | License URL | License" > THIRD-PARTY-LICENSES.md; echo "---|---|---" >> THIRD-PARTY-LICENSES.md; for i in $$(cat go.sum  | awk '{print $$1}'); do l=$$(go-licenses csv $$i 2>/dev/null); if [ $$? -ne 0 ]; then continue; fi; echo $$l | tr \, \| | tr ' ' '\n'; done | sort -u >> THIRD-PARTY-LICENSES.md)
+	go install github.com/google/go-licenses@latest
+	$(shell echo "Module | License URL | License" > THIRD-PARTY-LICENSES.md; echo "---|---|---" >> THIRD-PARTY-LICENSES.md; for i in $$(go list -m all  | awk '{print $$1}'); do l=$$(go-licenses csv $$i 2>/dev/null); if [ $$? -ne 0 ]; then continue; fi; echo $$l | tr \, \| | tr ' ' '\n'; done | sort -u >> THIRD-PARTY-LICENSES.md)
+
+.PHONY: check-licenses
+check-licenses:
+	go install github.com/google/go-licenses@latest
+	@for mod in $$(go list -m -f '{{if not (or .Indirect .Main)}}{{.Path}}{{end}}' all); do \
+		while [ x$$mod != x ]; do \
+			echo -n "Checking $$mod ... "; \
+			result=$$(GOFLAGS="-tags=extended,containers_image_openpgp" go-licenses check $$mod 2>&1); \
```

Comment:
I would check for 'minimal' tag too (just in case). We can add another for loop before while to iterate between "extended" and "minimal" tags.